test/test_dbgfile.py

```diff
@@ -0,0 +1,40 @@
+import unittest
```

Comment:
Please add a comment here explaining the purpose of this test - what specific functionality is it testing for